### DISCUSSION TITLE
ETA for HTTP metrics release

### DISCUSSION THREAD

**Discussion by EnDimeonRus** [open-telemetry/opentelemetry-dotnet#3941] [General]
Hi guys,
We are waiting for changes in Http metrics described here **https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md** in Unreleased section. Could you tell me when this changes will be available in nuget?

**Comment by vishweshbankwar** [comment]
There is a release planned this week.
